src/user/header/component.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Grid, Paper } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useStyle } from 'user/header/style'
+
+export const Components = ({ items }) => <Grid item xs={3}>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125620220-0a32d088-7fd1-4292-bd04-bc9e057eac2a.png)

1. The file is named `component.jsx`, so why is the component itself named `Components`?
2. On that note, is the name `Component` really a good naming practice? I mean, aren't them all components? :smile:

Reply:
Done.